packages/manager/modules/core/src/translate/translate.factory.js

```diff
@@ -1,13 +1,22 @@
+import get from 'lodash/get';
+import startsWith from 'lodash/startsWith';
+
 import {
   HEADER_LOCALE,
 } from './translate.constants';
 
 export default /* @ngInject */ TranslateService => ({
-  request: config => ({
-    ...config,
-    headers: {
-      ...config.headers,
-      [HEADER_LOCALE]: TranslateService.getUserLocale(),
-    },
-  }),
+  request: (config) => {
+    const headerConfig = {
+      ...config,
+      headers: {
+        ...config.headers,
+      },
+    };
+    // do not set content-language header for CORS requests
+    if (!startsWith(get(headerConfig, 'url'), 'https://')) {
```

Comment:
Why not allowing the parameter only on trusted routes like we did for `request-tagger` (https://github.com/ovh/manager/blob/master/packages/manager/modules/request-tagger/src/factory.js#L23 && https://github.com/ovh/manager/blob/master/packages/manager/modules/request-tagger/src/constants.js#L8-L12) ?